--- Running numpy.genfromtxt#242YY/genfromtxtYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'numpy.genfromtxt#242YY'), ('notebook', 'genfromtxtYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.432s
user	0m1.583s
sys	0m5.328s
